render.py

```diff
@@ -18,12 +18,12 @@
 import shutil
 import atexit
 
-#VERSION = "13.2"
-#RELEASE = datetime.datetime(2014, 11, 04, 12, 0, 0)
+VERSION = "13.2"
+RELEASE = datetime.datetime(2014, 11, 04, 12, 0, 0)
 
-VERSION = "conference"
+#VERSION = "conference"
```

Comment:
instead of commenting out the candidate values of VERSION, how about to write an example on a comment? Something like `VERSION should be a release number eg 13.2, or 'conference'`